pkg/keymanagementprovider/config/crl_config.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+type CRLConfig struct {
+	Type        string            `json:"type,omitempty"`
+	EnableCache bool              `json:"enable_cache,omitempty"`
+	Refreshable bool              `json:"refreshable,omitempty"`
+	HTTPRequest HTTPRequestConfig `json:"http_request,omitempty"`
```

Comment:
I would prefer keeping the config suffix as request and requestConfig are quite different.

Reply:
Sure.